test-app/app/templates/application.hbs

```diff
@@ -1,3 +1,45 @@
 <h2 id="title">Welcome to Ember</h2>
 
-{{outlet}}
\ No newline at end of file
+{{!-- template-lint-disable require-input-label --}}
```

Comment:
<img width="592" alt="Screen Shot 2023-02-27 at 2 36 30 PM" src="https://user-images.githubusercontent.com/771170/221666174-c63cc846-c0e6-4beb-9344-e3afd1e8741d.png">

Just note: I added this, but did not install the related styles. If we need that maybe it can be a follow up?